android/expoview/src/main/java/host/exp/exponent/notifications/NotificationHelper.java

```diff
@@ -575,19 +575,8 @@ public static void scheduleLocalNotification(
     final Listener listener) {
 
     HashMap<String, java.io.Serializable> details = new HashMap<>();
-
     details.put("data", data);
-
-    String experienceId;
-
-    try {
-      // needs to be stable legacy ID since previous local notifications were created using that format
-      experienceId = experienceKey.getStableLegacyId();
-      details.put("experienceId", experienceId);
-    } catch (Exception e) {
-      listener.onFailure(new Exception("Requires Experience Id"));
-      return;
-    }
+    details.put("experienceId", experienceKey.getScopeKey());
```

Comment:
Use `NotificationConstants` for the map keys? (We should verify whether the receiving side that reads from `details` uses `NotificationConstants` so we maintain symmetry.)